src/packages/badge/badge.taro.tsx

```
@@ -66,41 +77,75 @@ export const Badge: FunctionComponent<Partial<BadgeProps>> = (props) => {
+    }
+    setContentStyle(style)
+  }
 
+  const getStyle = () => {
+    const style: CSSProperties = {}
     if (color) {
       if (fill === 'outline') {
         style.color = color
-        style.background = '#fff'
+        isHarmony
+          ? (style.backgroundColor = '#fff')
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)